test/infinite_list_test.dart

```diff
@@ -21,57 +20,6 @@ void main() {
   group('InfiniteList', () {
     void emptyCallback() {}
 
-    testWidgets(
```

Comment:
why did we remove this test?

Reply:
Because scroll controlled does pretty much nothing now.

>  Breaking: Any ScrollController passed to InfiniteList is now merely a passthrough property. Any implementation or test that expects it to have a listener on InfiniteList will break.

Reply:
haha ah I see...makes sense 👍